src/include/common.h

```diff
@@ -98,7 +98,9 @@ typedef struct expresslrs_rf_pref_params_s
     uint32_t RxLockTimeoutMs;       // Max time to go from tentative -> connected state on receiver (ms)
     uint32_t SyncPktIntervalDisconnected; // how often to send the SYNC_PACKET packet (ms) when there is no response from RX
     uint32_t SyncPktIntervalConnected;    // how often to send the SYNC_PACKET packet (ms) when there we have a connection
-
+#if defined(Regulatory_Domain_ISM_2400)
+    uint32_t InstantRssiValidDelayUs;     // How long to wait for valid GetRssiInst() reading after enabling RX, worst case mode switch from TX to RX
```

Comment:
We should add an addition comment and reference here to the standard.  If we change anything it will be easy to find and update the number.

Reply:
In the latest commit I ended up not using this one at all, this timing is guaranteed by choice of hwtimer timeout and lora on air time. But yes, that should be documented somewhere.  

If this approach to LBT ends up looking feasible and gets consensus, I'll spend some more time on cleaning up, documenting, putting things were they belong etc. (a bit too many things in tx-main as it is now maybe, and we probably want the same scheme in rx-main for telemetry?